api/src/neo4j/driver.js

```diff
@@ -1,6 +1,9 @@
 import neo4j from 'neo4j-driver';
 
 const uri = 'bolt://neo4j';
+// To access on host computer (i.e. outside Docker container) use below uri
```

Comment:
Should such information be written here or in the wiki page? This introduced another way of testing compared to that is documented on the README.

Reply:
Don't know, maybe the best would be get it into a config file and then you could use then use that config for running API tests but that was a big bigger. If you do not want this info here then I will remove it.

Reply:
Maybe we can create a new issue to add instructions for testing outside of docker container (if there is such requirement). I just feel a bit unclear with the comments here.

Reply:
Will remove the comments then!